src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1087,6 +1087,17 @@ public override TypeInfo GetTypeInfo(SelectOrGroupClauseSyntax node, Cancellatio
         private void GetBoundNodes(CSharpSyntaxNode node, out CSharpSyntaxNode bindableNode, out BoundNode lowestBoundNode, out BoundNode highestBoundNode, out BoundNode boundParent)
         {
             bindableNode = this.GetBindableSyntaxNode(node);
+
```

Comment:
It looks like `MemberSemanticModel.GetBindableSyntaxNode` just walks the parent chain. Are there cases where the root is not the top-most node?